test/unit-test-http/CMakeLists.txt

```diff
@@ -0,0 +1,88 @@
+# Include filepaths for source and include.
```

Comment:
We already have one under `test/unit-test/CMakeLists.txt`, we duplicating this file?

Reply:
Agreed, changed structure in https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/62/commits/8f2a5031f8d914cbd8690c3cfe4c3c3ecc817782